.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "themes/docsy"]
 	path = themes/docsy
 	url = https://github.com/google/docsy.git
+[submodule "api-reference"]
+	path = api-reference
```

Comment:
nit: You may want to name this path something more specific. Some quick ideas:
`api-reference-submodule` or `api-ref-generator`

This is the download directory for the "generator" tool source code?

Reply:
@feloy @kbhawkey I do like `api-ref-generator` as a submodule name. 👍

Reply:
I have made the change